apollo-api/src/main/java/com/apollographql/apollo/api/Error.kt

```diff
@@ -0,0 +1,74 @@
+package com.apollographql.apollo.api
+
+/**
+ * Represents an error response returned from the GraphQL server
+ */
+data class Error(
+    /**
+     * Server error message
+     */
+    @JvmField
+    val message: String,
+
+    /**
+     * Locations of the errors in the GraphQL operation
+     */
+    @JvmField
+    val locations: List<Location> = emptyList(),
+
+    /**
+     * Custom attributes associated with this error
+     */
+    @JvmField
+    val customAttributes: Map<String, Any?> = emptyMap()
+) {
+
+  /**
+   * Returns server error message.
+   */
+  @Deprecated(message = "Use property instead", replaceWith = ReplaceWith(expression = "message"))
+  fun message(): String? = message
```

Comment:
+1 for the notice 👍 
I wonder how that's going to play with Java if we decide to remove `@JvmField` as mentioned above.

Reply:
On Java as far as I can see it just marked as deprecated without replace action, just a message. I think we are good here.